_posts/2016-09-02-homebrew-1.0.0.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 1.0.0
+author: MikeMcQuaid
+---
+Today I'm proud to announce Homebrew 1.0.0. In the seven years since Homebrew was created by [@mxcl](https://github.com/mxcl) our community has grown to [almost 6000 unique contributors](https://github.com/Homebrew/homebrew-core/graphs/contributors), a [wide-reaching third-party "tap" ecosystem](https://github.com/search?p=2&q=homebrew-&type=Repositories&utf8=✓) and [thousands of packages](https://github.com/Homebrew/homebrew-core/tree/master/Formula).
+
+We've been working hard over the last year to make some major changes to Homebrew that we've been wanting for a long time. There have been some hiccups along the way but we now have a more stable base for using and developing Homebrew in future.
+
+These include:
+
+- [Homebrew package management and formulae (package metadata) split into separate repositories](https://github.com/Homebrew/brew/pull/2)
+- [Homebrew Community site (using Discourse) for discussion](http://community.brew.sh)
+- [Homebrew joined the Software Freedom Conservancy](https://sfconservancy.org/news/2016/feb/22/homebrew-joins/)
+- [Homebrew's default repository installation location changed to `/usr/local/Homebrew` to keep your `/usr/local` cleaner](https://github.com/Homebrew/install/pull/60)
+- [Homebrew/brew updates between release tags](https://github.com/Homebrew/brew/pull/998)
+- [Homebrew CI and homebrew/core use the OS X sandbox for build-time security](https://github.com/Homebrew/legacy-homebrew/pull/38361)
```

Comment:
In hindsight, given we're using `macOS` almost everywhere in code, we probably should have used it here as well. Not a big deal.